cli/src/commands/extensions/api.rs

```diff
@@ -512,25 +512,23 @@ async fn api_base_url(op_state: Rc<RefCell<OpState>>) -> Result<String> {
     Ok(url)
 }
 
-pub(crate) fn api_decls() -> Vec<OpDecl> {
-    vec![
-        analyze::decl(),
-        check_packages::decl(),
-        check_packages_raw::decl(),
-        get_user_info::decl(),
-        get_access_token::decl(),
-        get_refresh_token::decl(),
-        get_job_status::decl(),
-        get_job_status_raw::decl(),
-        get_current_project::decl(),
-        get_groups::decl(),
-        get_projects::decl(),
-        create_project::decl(),
-        delete_project::decl(),
-        get_package_details::decl(),
-        parse_lockfile::decl(),
-        run_sandboxed::decl(),
-        op_permissions::decl(),
-        api_base_url::decl(),
-    ]
-}
+deno_core::ops!(api_decls, [
```

Comment:
Is there no non-macro based way to do this anymore?

I find it quite unfortunate to transform something so simple into an opaque macro nobody understands.